### PR TITLE
Car showroom 01

### PR DESCRIPTION
**Pull request by palakz-simform**
This is the Car Gallery Project where I have created a component gallery-card which contains information like name, image, description and price of the car.

### CODE REVIEW

File: car-showroom-01/src/App.vue [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+  border-radius: 5px;
+}
+.title h1 {
+  color: black;
+  font-size: 40px;
+  text-shadow: 0 0 3px gray;
+}
+
+@media (max-width: 800px) {
+  .car-card {
```
Comment (moksha-shah-simform) [inline car-showroom-01/src/App.vue]: remove same code

File: car-showroom-01/src/components/gallery-card.vue [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    border-radius: 10px;
+  }
+}
+
+@media (max-width: 670px) {
+  .card {
+    margin: 5px;
+    width: 175px;
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+    background-color: rgb(216, 215, 214);
```
Comment (moksha-shah-simform) [inline car-showroom-01/src/components/gallery-card.vue]: remove same code

File: car-showroom-01/src/components/gallery-card.vue [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+  .description {
+    height: 120px;
+    width: 175px;
+    text-align: center;
+  }
+  .btn {
+    width: 175px;
+    height: 50px;
+  }
+  .button {
```
Comment (moksha-shah-simform) [inline car-showroom-01/src/components/gallery-card.vue:179]: remove same code

**Review by Moksha-S — CHANGES_REQUESTED** [review]
- Add comment whenever it's required
- Try to use proper Naming convention
- Overall it's good

**Review by Moksha-S — APPROVED** [review]

**Pull request merged**